tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+- name: Prepare random number
+  set_fact:
```

Comment:
how about combine with mysql test as alias?

Reply:
well, actually I just fixed sku and readded the tests to preview modules, but I can still improve them ;-)